<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>484</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Обращение матриц</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridMenu">
     <item row="1" column="1">
      <widget class="QComboBox" name="method_box"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="method_label">
       <property name="text">
        <string>Метод:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="size_label">
       <property name="text">
        <string>Размер:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="size_line">
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>561</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridMatrix"/>
   </widget>
   <widget class="QPushButton" name="button_invert">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обратить матрицу</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_det">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>169</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Определитель матрицы = 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_freezeMatrix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>169</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрепить матрицу</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
